xml/s4s_setup_cluster.xml

```diff
@@ -656,6 +656,58 @@
    </step>
   </procedure>
  </sect1>
+ <sect1 xml:id="sec.s4s.hana-cluster.wizard.semiautomatic">
+  <title>Unattended Setup Using &hana;-SR Wizard</title>
+  <para>
+   Setting up a reproducible &hana; System Replication infrastructure
+   on several machines can be a lot of administrative work. The &hana;-SR
+   wizard is a manual tool that reduces the workload. However, it still
+   requires some manual steps by administrators to set it up.
+  </para>
+  <para>
+   &suse; is extending the &hana;-SR Wizard capabilities for the Azure Cloud.
+   Administrators can now use the &hana;-SR Wizard to configure the &hana;
+   cluster on Azure and only have to define a configuration inside the
+   wizard once and can then re-apply the saved configuration without
+   user interaction.
+  </para>
+  <para>
+   To deploy a preconfigured &hana;-SR environments, &suse; offers an
+   unattended installation mode for the &hana;-SR wizard.
+   To use it, proceed as follows:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Create a configuration file by running the <literal>sap_ha</literal>
+     &yast; module.
```

Comment:
I guess that needs a hint toward the manual setup section in this file.

Also not having tried this -- I wonder, do I have to fully set up HANA + HANA-SR locally before I can move my config to the cloud or how does it work? (If so, seems like a lot of effort just to move your deployment elsewhere immediately after.)